libs/testing/cypress/browser/src/lib/firestore.ts

```diff
@@ -26,7 +26,7 @@ export const firestore = {
     return cy.task('getData', [paths]).then(array => array[0]);
   },
 
-  queryData(data: { collection: string; field: string; operator: WhereFilterOp; value: unknown }) {
+  queryData<T>(data: { collection: string; field: string; operator: WhereFilterOp; value: unknown }): Cypress.Chainable<T> {
```

Comment:
I gave you a bad suggestion but in fact you dans do 
```suggestion
  queryData<T>(data: { collection: string; field: string; operator: WhereFilterOp; value: unknown }): Cypress.Chainable<T[]> {
```

Like this we can do ` firestore.queryData<Event>` (we are making a query on Event documents) and we return `Event[]` (firestore queries always return an array unless you specifically give and Id or a documentPath)